wp-modules/editor/model.php

```diff
@@ -126,10 +126,9 @@ function save_metadata_to_pattern_file( $override, $post_id, $meta_key, $meta_va
 			get_pattern_defaults(),
 			$pattern ? $pattern : [],
 			[
-				'name'  => $pattern_name,
-				'title' => $post->post_title,
```

Comment:
This is the fix: not saving the 'title', as this function is only for meta.

The 'title' is stored in the post, so the [post callback](https://github.com/studiopress/pattern-manager/blob/2cde28e1220ed40524dc292236f192af0eb187cb/wp-modules/editor/model.php#L66) should update it.